packages/atomic/src/components/common/query-summary/query-summary-common.tsx

```diff
@@ -4,7 +4,7 @@ import {AnyBindings} from '../interface/bindings';
 
 interface QuerySummaryCommonProps {
   bindings: AnyBindings;
-  enableDuration: boolean;
+  withDuration: boolean;
```

Comment:
This should be always true too and removed?

Reply:
It we want it to be possible to show the duration by using the `duration` part and `display: block`, then we need the duration to be rendered.

In `atomic-insight-query-summary`, we never display the duration, and it's not possible to show it with a part. Since this component is common across both, we need this option.

Reply:
Yes it should be always true and always possible to show using the part, I don't get why with insight's it isn't because the API doesn't return it?